.travis.yml

```diff
@@ -20,6 +21,8 @@ addons:
     - google-chrome-stable
 script:
   - brakeman --run-all-checks --exit-on-warn --format plain .
-  - bundle exec rspec spec
+  - npx percy exec -- bundle exec rspec spec
 after_script:
   - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
+PERCY_TOKEN:
```

Comment:
@Br3nda Should this key be in the Travis env instead of hard-coded here?

Reply:
If it's in Travis, it isn't available in PRs from forks.